src/content-helper/editor-sidebar/tabs/sidebar-performance-tab.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { Panel, PanelBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { SidebarSettings, useSettings } from '../../common/settings';
+import { VerifyCredentials } from '../../common/verify-credentials';
+import { PerformanceDetails } from '../performance-details/component';
+
+/**
+ * SidebarPerformanceTab component props.
+ *
+ * @since 3.14.0
+ */
+type SidebarPerformanceTabProps = {
+	trackToggle: ( panel: string, next: boolean ) => void
+}
+
+/**
+ * SidebarPerformanceTab component.
+ * Renders the Performance tab in the Content Helper sidebar.
+ *
+ * @since 3.14.0
+ *
+ * @param { SidebarPerformanceTabProps } props The component's props.
+ */
+export const SidebarPerformanceTab = ( { trackToggle }: SidebarPerformanceTabProps ) => {
+	const { settings, setSettings } = useSettings<SidebarSettings>();
+
+	return (
+		<Panel>
+			<PanelBody
+				title={ __( 'Performance Details', 'wp-parsely' ) }
+				initialOpen={ settings.PerformanceDetailsOpen }
+				onToggle={ ( next ) => {
```

Comment:
I've addressed this with a new commit. There's no margin so it doesn't look great, but since the inner component will be receiving a revamp, I've left it this way for now.